specification/appendix_d.adoc

```diff
@@ -0,0 +1,88 @@
+[[appendix_d]]
+== Appendix D: M-mode TSM based deployment model
+
+This deployment model targets high-assurance systems whose design might be constrained 
+by real-time and formal verification requirements. It trade offs a feature-rich design supporting 
+dynamic resource allocation, for simpler implementation, where the correctness can be formally verified.
+
+[id=depd]
+[caption="Figure {counter:image}"]
+[title= ": M-mode TSM based deployment model for CoVE"]
+image::img_11.png[align=center]
+
+=== Overview
+<<depd>> shows that the deployment model supports a single confidential supervisor domain in which 
+the TSM runs along with the TSM driver in the M-mode. This single confidential supervisor domain can run multiple 
+TVMs that are isolated from each other using the MMU, i.e., G-stage page tables managed by TSM. TSM isolates the 
+hosting supervisor domain (i.e., OS/VMM and non-confidential applications and VMs) from the confidential supervisor 
+domain (TSM and TVMs) using a hardware memory isolation mechanism, like PMP. The Smmtt extension is therefore not required. 
+IO accesses to confidential memory must be prevented, for example, with IOPMP.
+
+[NOTE]
+====
+Since the TSM is not required to run in the HS-mode, this deployment model supports systems that emulate the 
+hypervisor extension or run TVMs and OS/VMM in S-mode. The latter requires use of other mechanism to enforce 
+memory accesses to confidential memory, e.g., PMP. 
+====
+
+=== Static memory partitioning
+The deployment model proposes static partitioning of memory into confidential and non-confidential to simplify 
+formal reasoning about the correctness of the TVM execution and isolation. TSM performs this paritioning early 
+during the boot of the platform, resulting in the following advantages: (1) simplified formal reasoning about the 
+ownership of memory, (2) attestation that covers static system configuration (e.g., values of PMP registers), 
+(3) reduced attack surface between OS/VMM and TSM (narrower ABI). A possible negative consequence of 
+static partitioning is underutilization of resources.
```

Comment:
perhaps better to clarify - underutilization of memory resources specifically since dynamic page-based conversion of memory between confidential and hosting supervisor domains is not possible in this scenario (unless Smmtt is used).

Reply:
I added the clarification of underutilization of memory